dev/dev-scenarios.nix

```diff
@@ -45,4 +45,34 @@ with lib;
     nix-bitcoin.nodeinfo.enable = true;
     # test.container.enableWAN = true;
   };
+
+  wireguard-lndconnect-online = { config, pkgs, lib, ... }: {
+    imports = [
+      ../modules/presets/wireguard.nix
+      scenarios.regtestBase
+    ];
+
+    # 51820 (default wg port) + 1
+    networking.wireguard.interfaces.wg-nb.listenPort = 51821;
+    test.container.enableWAN = true;
+    # test.container.exposeLocalhost = true;
+
+    services.clightning.extraConfig = "disable-dns";
+
+    services.lnd = {
+      enable = true;
+      lndconnect = {
+        enable = true;
+        onion = true;
+      };
+    };
```

Comment:
Shouldn't I expect to see an lnd-rest entry in nodeinfo?
```
dev-run-tests -s wireguard-lndconnect-online container --run c nodeinfo
Building container
/nix/store/ncgfyk9r7hqxnxvs8k2hzsc716v16b3l-container

Installing containers:
nb-test

Starting containers:
nb-test

Running command.
{
  "bitcoind": {
    "local_address": "127.0.0.1:8333"
  },
  "clightning": {
    "local_address": "127.0.0.1:9735",
    "nodeid": "030cb0ec1c10551a0527187af5bbf83afdee5eefde404aca85c18364e3a902609d"
  },
  "clightning-rest": {
    "local_address": "127.0.0.1:3001",
    "onion_address": "n3ahtp4ariczami6tllmpoofulmf43efxtfm6d4tewqf3h7qeekyrjid.onion:3001"
  },
  "lnd": {
    "local_address": "localhost:9736",
    "onion_address": "zrdmpcmxfwsrc2ggjxqdqizp72ivxvno3jca33hyxadxmq7gykvcrxyd.onion:9736",
    "nodeid": "03e01e8ed3810d5a50958392de464986994e4f29a08d2e7f80ef134f020bd81a08"
  }
}
```

Reply:
Fixed in `nodeinfo/lnd: add `rest_address``.

Reply:
Shouldn't this also output the onion (if configured)?

Reply:
Fixed in `nodeinfo/lnd: add onion_rest_address`.